src/clients/twitch.ts

```diff
@@ -12,128 +15,111 @@ function registerTwitch(): void
 {
     /**
     * Login to twitch using the access token found in our oauth process
-    * @param {string} {access_token} a javascript object containing at minimum, a twitch access token 
-    * @param {boolean} skipTokenSave
     */
-    function loginToTwitch({ access_token }: { access_token: string; }, skipTokenSave: boolean): void
+    async function loginToTwitch(): Promise<void>
     {
         //TODO: detect replies by looking at previously recorded messages (userState["reply-parent-msg-id"])
-        //If we have the save token flag enabled, save this to "devTwitchToken"
-        if(!skipTokenSave && configFile.T2D_DEV_SAVE_TOKEN)
-            fs.writeFile('./devTwitchToken', access_token, null, error =>
-            {
-                if(error) console.error('I hit a snag...', error);
+        let tokenData: { accessToken: string, expiresIn: number, refreshToken: string, scope: string[], tokenType: string };
+
+        try
+        {
+            tokenData = JSON.parse(await fs.readFile('./tokens.json', 'utf-8'));
```

Comment:
As far as I know, utf-8 is the default, so it's why I left it out initially :+1:

Reply:
That's just how Twurple does it in their docs, that was basically ripped from there. (:

Reply:
Actually! Looking at it, we have to pass utf-8 or else FS thinks we're passing in a buffer instead of a path! I love Node.